src/Microsoft.Identity.Client/AppConfig/AadAuthorityAudience.cs

```diff
@@ -36,6 +38,7 @@ public enum AadAuthorityAudience
         /// <summary>
         /// The sign-in audience was not specified
```

Comment:
Please add a comment on why this member is hidden

Reply:
This doesn't actually work (it doesn't hide the enum value from intellisense).  So i'm going to revert this.

---
In reply to: [251772258](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/844#discussion_r251772258) [](ancestors = 251772258)